src/libertem/web/server.py

```diff
@@ -100,7 +101,30 @@ def make_app(event_registry, shared_state):
             "state": shared_state,
             "event_registry": event_registry,
         }),
-    ], **settings)
+        (r"/api/session/", SessionHandler, {
```

Comment:
These are unused and I will remove them in the next commit. I had to deal with stashing client changes and docker testing changes every time I commit. That's why this slipped in here. Don't worry about these two routes. I don't use them.